src/scss/next.scss

```diff
@@ -282,7 +333,7 @@ a:active {
 
 /// General interactive states
 
-:focus {
+:focus-visible {
```

Comment:
🙌

For webkit are you thinking we'd use the default browser focus state?

Reply:
You knows it. Good ol' progressive enhancement